docs/index.mdx

```diff
@@ -46,6 +46,22 @@ Second, a GitPOAP is awarded - Contributors are awarded GitPOAPs (POAPs with an
 
 <HowSvg />
 
+### Building on GitPOAP
+
+On-chain identities are only useful if there are others to interact with and applications built to leverage them.  That’s why we decided to build on top of POAP.  
+
+GitPOAP is meant to be a platform that will enable a new era of web3 applications, and we’re excited to see what else people build on top of it.  Our public API is [now live](https://docs.gitpoap.io/api)!  Start building and feel free to reach out if there’s anything we can do to better support you.
+
+### Why is this important?
+
+We’re building GitPOAP to solve real-world challenges we’ve faced while bootstrapping and growing open source projects.  web3 provides new modes of interaction and we believe GitPOAP can enable a novel way to reward contributors based on their value creation.  We believe this model can be generalized beyond open source and also enable new paradigms of collaboration.
```

Comment:
capital W for web3? 

`Web3 provides new modes of interaction and GitPOAP enables a novel way to reward contributors based on the value they add.`

Reply:
I've seen web3 and Web 2.0 spelled each that way in many places, so I think i'm going to leave it if that's cool?

Reply:
I made the other change tho 💪